doc/man3/EVP_PKEY_meth_new.pod

```diff
@@ -369,6 +374,9 @@ object or returns NULL if not found.
 EVP_PKEY_meth_add0() returns 1 if method is added successfully or 0
 if an error occurred.
 
+EVP_PKEY_meth_remove() returns 1 if method is removed successfully or
+0 if the method has not been added previously.
```

Comment:
I'd prefer to see: `0 if an error occurred.` here.

I realize the only error possible at the moment is if the item isn't in the stack but that might not be true forever.